ci/driver.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash --login
+#
+#####################################################################################
+#
+# Script description: Top level driver script for checking PR
+#                     ready for CI regresssion testing
+#
+# Abstract:
+#
+#  This script uses GitHub CL to check for Pull Requests with {machine}-CI tags
+#  on the development branch for the global-workflow repo and stages tests directors
+#  per PR number and then calling run_ci.sh to clone and  building from
+#  $(HOMEgfs)/sorc and then run a suite of regression tests with various configurations.
+#
+#######################################################################################
+
+#################################################################
+# TODO using static build for GitHub CLI until fixed in HPC-Stack
+#################################################################
+GH=/home/Terry.McGuinness/bin/gh
+
+################################################################
+# Setup the reletive paths to scripts and PS4 for better logging 
+################################################################
+set -eux
+pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+scriptname=$(basename "${BASH_SOURCE[1]}")
+start_time=$(date +%s)
+start_time_human=$(date -d"@${start_time}" -u)
+echo "Begin ${scriptname} at ${start_time_human}"
```

Comment:
Yes, I had, I took it out because Rahul asked me not to use any dependencies on the repo for two main BASH scripts so anyone can use them to pull PR and stage tests as a general use case.